package.json

```diff
@@ -58,12 +68,11 @@
     "prebench": "npm run --silent build:cjs",
     "bench": "node scripts/benchRunner",
     "bookmarklet": "node scripts/bookmarklet",
-    "build:es": "cross-env BABEL_ENV=es babel source --out-dir es",
+    "build:es": "cross-env BABEL_ENV=es babel source --out-dir es && node ./scripts/addModulePackageScope.js",
     "build:cjs": "cross-env BABEL_ENV=cjs babel source --out-dir src",
-    "build:mjs": "cross-env BABEL_ENV=es babel source/index.js --out-file src/index.mjs",
```

Comment:
I don't think this was particularly needed and should be even more obsolete thanks to the changes here.